src/proxy-ui-api/frontend/src/views/InitialConfiguration/ConfigurationAnchorStep.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div>
+    <div class="action-row">
+      <div>{{ $t('initialConfiguration.anchor.info') }}</div>
+      <upload-configuration-anchor-dialog
+        @uploaded="fetchConfigurationAnchor"
+        initMode
+      />
+    </div>
+
+    <template v-if="configuratonAnchor">
+      <div class="row-wrap">
+        <div class="label">{{ $t('initialConfiguration.anchor.hash') }}</div>
+        <template v-if="configuratonAnchor">{{
+          configuratonAnchor.hash | colonize
+        }}</template>
+      </div>
+
+      <div class="row-wrap">
+        <div class="label">
+          {{ $t('initialConfiguration.anchor.generated') }}
+        </div>
+        <template v-if="configuratonAnchor">{{
+          configuratonAnchor.created_at | formatDateTime
+        }}</template>
+      </div>
+    </template>
+    <div v-else style="height: 120px;"></div>
```

Comment:
What does this do? Couldn't this be done as an optional html-class & margin?

Reply:
Idea was to keep the height same with or without anchor data, but it's not a good implementation. Fixed now.